### PR TITLE
Make sure ilab wrapper is executable and in path

### PR DESCRIPTION
**Pull request by javipolo**

### CODE REVIEW

File: training/nvidia-bootc/Containerfile [hunk 1]
```diff
@@ -165,7 +165,9 @@ RUN grep -q /usr/lib/containers/storage /etc/containers/storage.conf || \
 	  cp /run/.input/ilab /usr/local/bin/ilab; \
      else \
 	  curl -o /usr/local/bin/ilab "https://raw.githubusercontent.com/containers/ai-lab-recipes/main/training/ilab-wrapper/ilab"; \
-    fi
+    fi \
+    && chmod +x /usr/local/bin/ilab \
+    && echo 'PATH=$PATH:/usr/local/bin' > /etc/profile.d/usrlocalbin.sh
```
Comment (Gregory-Pereira) [inline training/nvidia-bootc/Containerfile]: I was under the assumption `/usr/local/bin` is in the default `$PATH` so when the OS is installed via the `bootc` container it should be there. Is this assumption incorrect?

Comment (javipolo) [reply training/nvidia-bootc/Containerfile]: It is for users, but not for root, and IIRC ilab should be run as root

```
[cloud-user@javipolo-rhel-ai-0705a ~]$ echo $PATH
/var/home/cloud-user/.local/bin:/var/home/cloud-user/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin

[cloud-user@javipolo-rhel-ai-0705a ~]$ sudo -i
[root@javipolo-rhel-ai-0705a ~]# echo $PATH
/usr/local/sbin:/sbin:/bin:/usr/sbin:/usr/bin
```

Comment (rhatdan) [reply training/nvidia-bootc/Containerfile]: Why not install ibal in /usr/bin/  ?usr/local should not be used.

Comment (javipolo) [reply training/nvidia-bootc/Containerfile]: I just changed the commit to copy to `/usr/bin` and stop using /usr/local/bin

**Review by rhatdan — APPROVED** [review]
LGTM

**Pull request merged**